powerfulseal/policy/action_clone.py

```diff
@@ -189,12 +189,31 @@ def update_labels(**kwargs):
     return True
 
   def mutate_traffic_control(self, body, spec):
-    """ Adds an init container with tc """
+    """ Adds an init container with tc setup """
     if body.spec.template.spec.init_containers is None:
       body.spec.template.spec.init_containers = []
+    setup_spec = spec.get("setup")
     body.spec.template.spec.init_containers.append(
       kubernetes.client.V1Container(
-        name="chaos"+str(1+len(body.spec.template.spec.init_containers)),
+        name="chaos-setup-"+str(1+len(body.spec.template.spec.init_containers)),
```

Comment:
Shouldn't this be optional? The scheme doesn't require the setup step, so we should probably allow starting without it too.